blockbase/inc/customizer/wp-customize-fonts.php

```diff
@@ -209,13 +209,61 @@ class GlobalStylesFontsCustomizer {
 	);
 
 	function __construct() {
+		if ( $this->site_editor_is_implementing_google_fonts() ) {
+			add_action( 'customize_register', array( $this, 'display_section_redirecting_users_to_site_editor' ) );
+			return;
+		}
+
 		add_action( 'customize_register', array( $this, 'initialize' ) );
 		add_action( 'customize_preview_init', array( $this, 'handle_customize_preview_init' ) );
 		add_action( 'customize_register', array( $this, 'enqueue_google_fonts' ) );
 		add_action( 'customize_save_after', array( $this, 'handle_customize_save_after' ) );
 		add_action( 'customize_controls_enqueue_scripts', array( $this, 'customize_control_js' ) );
 	}
 
+	function site_editor_is_implementing_google_fonts() {
+		$jetpack_has_google_fonts_module = false;
+		$gutenberg_webfonts_api_supports_enqueueing = false;
+
+		if ( defined( 'JETPACK__VERSION' ) ) {
+			$jetpack_has_google_fonts_module = JETPACK__VERSION === 'wpcom' || version_compare( JETPACK__VERSION, '10.8', '>' );
+		}
+
+		if ( defined( 'GUTENBERG_VERSION' ) ) {
+			$gutenberg_webfonts_api_supports_enqueueing = version_compare( GUTENBERG_VERSION, '13.0', '>=' );
+		}
+
+		return $jetpack_has_google_fonts_module && $gutenberg_webfonts_api_supports_enqueueing && Jetpack::is_module_active( 'google-fonts' );
```

Comment:
**Note:** Jetpack::is_module_active always returns true for dotcom simple regardless of what value is passed to it. Unsure why at the moment

Reply:
@jeyip yup that is by design. From a code comment on `WPCOM_Site::is_module_active`:

> Always returns true. In Jetpack context it would actually check if a module is active or not because the module loading relies on the 'jetpack_active_modules' option. Here in WordPress.com context however, we always load everything, so in order to enable or disable something we have to check each module's settings.